.ncurc.js

```diff
@@ -12,7 +12,14 @@
   - next
     - 13.4.13 causes failures with fetch
 */
-const pinned = ["styled-components", "date-fns", "date-fns-tz", "eslint"]
+const pinned = [
+  "styled-components",
+  "date-fns",
+  "date-fns-tz",
```

Comment:
We don't need this now!

* https://github.com/marnusw/date-fns-tz/releases/tag/v3.0.0